### PR TITLE
Replacing COMMAND with CLAUSE

### PR DESCRIPTION
**Pull request by singlarohit**
I am proposing to change the term "command" with "clause" as they are used interchangeably throughout the episodes  2, 3 and 4 of the lesson. I believe that only one term (preferably "clause") must be used to avoid any confusion to the readers. 

Because SQL comes from the field of Computer science, I believe that use of "clause" is more appropriate in this context. I strongly feel that the above occurrences of "command" must be changed with "clause" to maintain consistency in the lesson.

### CODE REVIEW

File: _episodes/01-sql-basic-queries.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ right after SELECT:
 
 ### Limiting results
 
-Sometimes you don't want to see all the results, you just want to get a sense of what's being returned. In that case, you can use the `LIMIT` command. In particular, you would want to do this if you were working with large databases.
+Sometimes you don't want to see all the results, you just want to get a sense of what's being returned. In that case, you can use the `LIMIT` clause. In particular, you would want to do this if you were working with large databases.
```
Comment (jd-foster) [inline _episodes/01-sql-basic-queries.md:45]: Agreed, but change "the" to "a" : "a `LIMIT` clause`

File: _episodes/02-sql-aggregation.md [hunk 2]
```diff
@@ -241,10 +241,10 @@ we might get tripped up:
     FROM summer_2000
     WHERE species_id = 'PE';
 
-Here the `COUNT` command includes all five records (even those with NULL
+Here the `COUNT` clause includes all five records (even those with NULL
```
Comment (jd-foster) [inline _episodes/02-sql-aggregation.md:244]: Here, "function" seems the best choice

File: _episodes/02-sql-aggregation.md [hunk 3]
```diff
@@ -241,10 +241,10 @@ we might get tripped up:
...
 values), but the `SUM` only includes the three records with data in the
 `weight` field, giving us an incorrect average. However,
-our strategy *will* work if we modify the `COUNT` command slightly:
+our strategy *will* work if we modify the `COUNT` clause slightly:
```
Comment (jd-foster) [inline _episodes/02-sql-aggregation.md:247]: Also "function", but "clause" works if you feel strongly about it.

File: _episodes/03-sql-joins.md [hunk 4]
```diff
@@ -9,27 +9,27 @@ objectives:
 - "Apply functions to manipulate individual values."
 - "Employ aliases to assign new names to tables and columns in a query."
 keypoints:
-- "Use the `JOIN` command to combine data from two tables---the `ON` or `USING` keywords specify which columns link the tables."
-- "Regular `JOIN` returns only matching rows. Other join commands provide different behavior, e.g., `LEFT JOIN` retains all rows of the table on the left side of the command."
+- "Use the `JOIN` clause to combine data from two tables---the `ON` or `USING` keywords specify which columns link the tables."
```
Comment (jd-foster) [inline _episodes/03-sql-joins.md:12]: This one should be either "keyword" or change to "a `JOIN` clause".

File: _episodes/03-sql-joins.md [hunk 5]
```diff
@@ -9,27 +9,27 @@ objectives:
...
+- "Regular `JOIN` returns only matching rows. Other join clauses provide different behavior, e.g., `LEFT JOIN` retains all rows of the table on the left side of the clause."
```
Comment (jd-foster) [inline _episodes/03-sql-joins.md:13]: Yes.

File: _episodes/03-sql-joins.md [hunk 6]
```diff
@@ -9,27 +9,27 @@ objectives:
...
 - "`COALESCE` allows you to specify a value to use in place of `NULL`, which can help in joins"
 - "`NULLIF` can be used to replace certain values with `NULL` in results"
 - "Many other functions like `COALESCE` and `NULLIF` can operate on individual values."
 ---
 
 ## Joins
 
-To combine data from two tables we use the SQL `JOIN` command, which comes after
-the `FROM` command.
+To combine data from two tables we use the SQL `JOIN` clause, which comes after
```
Comment (jd-foster) [inline _episodes/03-sql-joins.md:21]: "a SQL `JOIN` clause"

File: _episodes/03-sql-joins.md [hunk 7]
```diff
@@ -9,27 +9,27 @@ objectives:
...
 Database tables are used to organize and group data by common characteristics or principles.  
 Often, we need to combine elements from separate tables into a single tables or queries for analysis and visualization.
 A JOIN is a means for combining columns from multiple tables by using values common to each.
 
-The JOIN command combined with ON is used to combine fields from separate tables.  
+The JOIN clause combined with ON is used to combine fields from separate tables.  
 
 
-The `JOIN` command on its own will result in a cross product, where each row in
+The `JOIN` clause on its own will result in a cross product, where each row in
```
Comment (jd-foster) [inline _episodes/03-sql-joins.md:32]: "A `JOIN` clause"

File: _episodes/03-sql-joins.md [hunk 8]
```diff
@@ -9,27 +9,27 @@ objectives:
...
+To combine data from two tables we use the SQL `JOIN` clause, which comes after
+the `FROM` clause.
 
 
 Database tables are used to organize and group data by common characteristics or principles.  
 Often, we need to combine elements from separate tables into a single tables or queries for analysis and visualization.
 A JOIN is a means for combining columns from multiple tables by using values common to each.
 
-The JOIN command combined with ON is used to combine fields from separate tables.  
+The JOIN clause combined with ON is used to combine fields from separate tables.
```
Comment (jd-foster) [inline _episodes/03-sql-joins.md:29]: "keyword"

File: _episodes/03-sql-joins.md [hunk 9]
```diff
@@ -46,8 +46,8 @@ species id.
 the `table.colname` format to tell the manager what column in which table we are
 referring to.
 
-The output of the `JOIN` command will have columns from the first table plus the
-columns from the second table. For the above command, the output will be a table
+The output of the `JOIN` clause will have columns from the first table plus the
```
Comment (jd-foster) [inline _episodes/03-sql-joins.md:49]: "The output from using the `JOIN` clause"

File: _episodes/03-sql-joins.md [hunk 10]
```diff
@@ -46,8 +46,8 @@ species id.
...
+columns from the second table. For the above clause, the output will be a table
```
Comment (jd-foster) [inline _episodes/03-sql-joins.md:50]: "statement"

File: _episodes/03-sql-joins.md [hunk 11]
```diff
@@ -136,11 +136,11 @@ survey data.
 This is because, by default, SQL only returns records where the joining value
 is present in the joined columns of both tables (i.e. it takes the _intersection_
 of the two join columns). This joining behaviour is known as an `INNER JOIN`.
-In fact the `JOIN` command is simply shorthand for `INNER JOIN` and the two
+In fact the `JOIN` clause is simply shorthand for `INNER JOIN` and the two
```
Comment (jd-foster) [inline _episodes/03-sql-joins.md:139]: "keyword"

File: _episodes/03-sql-joins.md [hunk 12]
```diff
@@ -136,11 +136,11 @@ survey data.
...
 terms can be used interchangably as they will produce the same result.
 
 We can also tell the computer that we wish to keep all the records in the first
-table by using the command `LEFT OUTER JOIN`, or `LEFT JOIN` for short.
+table by using the clause `LEFT OUTER JOIN`, or `LEFT JOIN` for short.
```
Comment (jd-foster) [inline _episodes/03-sql-joins.md:143]: "by using a `LEFT OUTER JOIN` clause"

**Review by jd-foster — CHANGES_REQUESTED** [review]
I've tried to use the words that best fit in the context.

**Comment by singlarohit** [comment]
I just committed the aforementioned changes (Commit # 063b8e9) suggested by you.

**Comment by jd-foster** [comment]
Thanks, very nice.

**Pull request merged**